src/test/java/uk/gov/hmcts/reform/mi/miextractionservice/component/encryption/PgpDecryptionHelper.java

```diff
@@ -55,12 +56,12 @@ public static String decryptFile(byte[] in, InputStream keyIn, char... passwd) t
         PGPEncryptedDataList encryptedDataList = getPgpEncryptedDataList(in);
 
         // find the secret key
-        Iterator<PGPPublicKeyEncryptedData> it = encryptedDataList.getEncryptedDataObjects();
+        Iterator<PGPEncryptedData> it = encryptedDataList.getEncryptedDataObjects();
```

Comment:
Does this change need to be tested?

Reply:
Could do, but the same code is actually already in mi-azure-functions and working. I copied it from here to there before and had to make this change with the latest bouncycastle version. Working for the v2 decryption atm.